packages/cli/src/commands/grandamento/propose.ts

```diff
@@ -0,0 +1,63 @@
+import { StableToken } from '@celo/contractkit'
+import { flags } from '@oclif/command'
+import { BaseCommand } from '../../base'
+import { newCheckBuilder } from '../../utils/checks'
+import { displaySendTx } from '../../utils/cli'
+import { Flags } from '../../utils/command'
+import { enumEntriesDupWithLowercase } from '../../utils/helpers'
+
+const stableTokenOptions = enumEntriesDupWithLowercase(Object.entries(StableToken))
+
+export default class Propse extends BaseCommand {
+  static description = 'Proposes a Granda Mento exchange'
+
+  static flags = {
+    ...BaseCommand.flags,
+    from: Flags.address({ required: true, description: 'The address with tokens to exchange' }),
+    value: Flags.wei({
+      required: true,
+      description: 'The value of the tokens to exchange',
+    }),
+    stableToken: flags.enum({
+      required: true,
+      options: Object.keys(stableTokenOptions),
+      description: 'Name of the stable to receive or send',
+      default: 'cUSD',
+    }),
+    sellCelo: flags.boolean({
```

Comment:
An interesting finding when testing this out: you can't do `--sellCelo true` or `--sellCelo false`, you simply do `--sellCelo` to indicate true, and omit it to indicate `false`. However because this is required, it complains when you don't use `--sellCelo`, so this effectively only allows proposals that sell CELO.
What do you think of making this not required, adding a `--buyCelo` flag, and having them be exclusive to each other with: `exclusive: ['<the-other-flag-name>']`?

Reply:
hmmm, interesthing find. I guess we can't make them exclusive because they should be flagged as required as well.

Reply:
I couldn't figure out how to make the parser to ask explicitly for the value with bools, so changed the flag to enum and parsed the true myself. Added some tests to make sure it works as expected